core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -788,30 +811,37 @@ func (fm *FluxMonitor) pollIfEligible(pollReq PollRequestType, deviationChecker
 	// Call the v2 pipeline to execute a new pipeline run
 	// Note: we expect the FM pipeline to scale the fetched answer by the same
 	// amount as precision
-	runID, answer, err := fm.pipelineRun.Execute(metaDataForBridge)
```

Comment:
Any reason why you got rid of the pipelineRun wrapper? You could just make it only call `Execute` and it will return the same result. It also has tests that don't seem to have been replaced.

Reply:
yeah I think with this change to separate execute + insert it wasn't provide much value as an abstraction. I could make it just call Execute, but because InsertFinishedRun needs the run and results, it would be pared down to an Execute which just calls another Execute and nothing else. Testwise - the only loss of coverage is on the ToDecimal error case which seemed ok to me